charts/nimbus/values.yaml

```diff
@@ -128,9 +128,9 @@ nimbus:
   # -- Pod-wide security context
   podSecurityContext:
     runAsNonRoot: true
-    runAsUser: 101337
-    runAsGroup: 101337
-    fsGroup: 101337
+    runAsUser: 1000
+    runAsGroup: 1000
+    fsGroup: 1000
```

Comment:
In newest version of nimbus the image has changed from using [`debian:bullseye-slim` to `debian:bookworm-slim` ](https://github.com/status-im/nimbus-eth2/pull/6740/files) which seems to break permissions with:
```
101337@nimbus-1-nimbus-0:/home/user$ ls -l nimbus-eth2/build
ls: cannot access 'nimbus-eth2/build': Permission denied
```